cli/main.c

```diff
@@ -90,6 +90,25 @@ int switchtec_handler(const char *optarg, void *value_addr,
 	return 0;
 }
 
+/*
+ * The 'mfg' submenu commands are only available in Linux builds.
+ *
+ * Due to the difference in driver architecture, supporting Windows
+ * build is non-trivial. After evaluating the development effort
+ * and the resources available, we have decided to remove Windows
+ * build support from release roadmap.
+ * 
+ */
+int mfg_handler(const char *optarg, void *value_addr,
+		const struct argconfig_options *opt)
+{
+#ifndef __linux__
+	fprintf(stderr, "Error: 'mfg' submenu commands are only supported in Linux build!\n");
+	return -1;
```

Comment:
I'd much rather see this as warning than a hard error. Seeing there's no technical reason they would not work.

Reply:
OK, updated to print a warning message and continue. @lsgunth please take a look.